components/Settings/FavoritesTab.js

```diff
@@ -1,9 +1,17 @@
+import { useState } from 'react'
 import ClubTableRow from '../ClubTableRow'
 
-export default (props) => {
-  const { updateFavorites, favorites, clubs } = props
-
+export default ({ clubs, favorites, updateFavorites }) => {
+  const [table, setTable] = useState(() => {
+    const ret = {}
+    favorites.forEach(favorite => { ret[favorite] = true })
```

Comment:
I think prettier would recommend this but I might be wrong
```suggestion
    favorites.forEach(favorite => {
      ret[favorite] = true
    })
```